avocado/plugins/spawners/common.py

```diff
@@ -0,0 +1,30 @@
+from avocado.core.plugin_interfaces import CLI
+from avocado.core.settings import settings
+
+
+class SpawnerCLI(CLI):
+
+    name = 'spawner'
+    description = 'spawner command line options for "run"'
+
+    def configure(self, parser):
+        super(SpawnerCLI, self).configure(parser)
+        parser = parser.subcommands.choices.get('run', None)
+        if parser is None:
+            return
+
+        parser = parser.add_argument_group('spawner specific options')
+        help_msg = ("This will disable the fetching output files from test "
```

Comment:
This wording gives me the impression setting it True will result in disabling the fetching, is that correct? I would suggest this reads: "This controls whether to fetch ..."